src/unstract/sdk/adapters/exceptions.py

```diff
@@ -9,13 +10,5 @@ class LLMError(AdapterError):
     pass
 
 
-class VectorDBError(AdapterError):
```

Comment:
Did we remove this because it is not being used anywhere?

Reply:
Ok, I see these being defined in unstract.sdk.exceptions. Should LLMError also be removed from here then?

Reply:
@gaya3-zipstack yes, to avoid confusion and to keep it simple we should aim to remove the other exceptions defined in this file. I've added a TODO for it since I noticed some usages. I'll try to remove those as well but I think its being used downstream as well so maybe we could plan it later too